conjurapi/variable_test.go

```diff
@@ -120,7 +121,7 @@ func TestClient_RetrieveSecret(t *testing.T) {
 					variableIds = append(variableIds, id)
 				}
 
-				secrets, err := conjur.RetrieveBatchSecrets(variableIds)
+				secrets, err := conjur.RetrieveBase64BatchSecrets(variableIds)
```

Comment:
can we still add more tests here to better cover the various permutations allowed? we still support a direct call to `RetrieveBatchSecrets`, right? so we probably shouldn't just drop testing it

Reply:
I added a new testing section which uses the new method for binary secrets and left the old tests as is